### ISSUE TITLE
Implement Dropout layer

### ISSUE THREAD

**Issue by wmeints** [wmeints/Neuromatic#8]
In order to help developers prevent overfitting of the neural network, implement support for the dropout layer type.
